coral/coral/settings.py

```diff
@@ -316,7 +316,7 @@ class GeoJSON:
 
 OAUTH_CLIENT_ID = ''  #'9JCibwrWQ4hwuGn5fu2u1oRZSs9V6gK8Vu8hpRC4'
 
-APP_VERSION = '0.3.2'
+APP_VERSION = '0.4.5'
```

Comment:
Make this match the version on develop to allow the auto updating to take affect